src/compas_model/elements/element.py

```diff
@@ -46,51 +50,59 @@ class Element(Data):
     """
 
     @property
-    def __data__(self) -> dict:
+    def __data__(self):
+        # type: () -> dict
         return {"geometry": self.geometry, "frame": self.frame, "name": self.name}
 
-    def __init__(self, geometry: Union[Geometry, Mesh] = None, frame=None, name=None):
-        super().__init__(name=name)
+    def __init__(self, geometry=None, frame=None, name=None):
+        # type: (compas.geometry.Geometry | compas.datastructures.Mesh | None, Frame | None, str | None) -> None
```

Comment:
Can the geometry input be a list? 
Example a polygon with holes or a pair of polygons.